connectors/src/resources/connector_resource.ts

```diff
@@ -0,0 +1,28 @@
+import type { ConnectorProvider } from "@dust-tt/types";
+import type { ModelStatic } from "sequelize";
+
+import { BaseResource } from "@connectors/resources/base_resource";
+import { ConnectorModel } from "@connectors/resources/storage/models/connector_model";
+import type { ReadonlyAttributesType } from "@connectors/resources/storage/types";
+
+// Attributes are marked as read-only to reflect the stateless nature of our Resource.
+// This design will be moved up to BaseResource once we transition away from Sequelize.
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ConnectorResource
+  extends ReadonlyAttributesType<ConnectorModel> {}
+export class ConnectorResource extends BaseResource<ConnectorModel> {
+  static model: ModelStatic<ConnectorModel> = ConnectorModel;
+
+  static async listByType(type: ConnectorProvider) {
```

Comment:
Is this used?

Reply:
It's used in a follow up PR to refactor the code from the migrations.